### PR TITLE
Order Copy Cards form view

### PR DESCRIPTION
**Pull request by cintamani**
From: https://eaflood.atlassian.net/browse/RUBY-767

This implements the view for the first page in the order copy cards flow, as per design.

Screenshot 2019-12-09 at 22 17 04

### CODE REVIEW

File: config/locales/forms/copy_cards_forms/en.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+en:
+  waste_carriers_engine:
+    copy_cards_forms:
+      new:
+        heading: "Order registration cards for %{reg_identifier}"
+        temp_cards_label: "Number of cards at £5 each"
+        next_button: "Continue"
+        check_the_address: "Check the address to post the cards to:"
```
Comment (Cruikshanks) [inline config/locales/forms/copy_cards_forms/en.yml]: We've got this bit of text, but I can't see it referenced in the view or in the screenshot in the PR description.

Should it be included somewhere?

Comment (cintamani) [reply config/locales/forms/copy_cards_forms/en.yml]: Oh. I have no idea of why it is there :/ I didn't even copy pasted this code! 😮 🤷‍♂ maybe some `sleepcoding`? 😆

Comment (cintamani) [reply config/locales/forms/copy_cards_forms/en.yml]:
```suggestion
```

Comment (Cruikshanks) [reply config/locales/forms/copy_cards_forms/en.yml]: Hey @cintamani I've gone back and checked the wireframe, and I think this is a case of the `.yml` was right, you have just missed it off the view.

Screenshot 2019-12-11 at 13 15 00

Unless you are working off a later wireframe ???

Comment (cintamani) [reply config/locales/forms/copy_cards_forms/en.yml]: 😮 🤷‍♀ Sorry! I think is just a distraction!

File: config/locales/forms/copy_cards_forms/en.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+en:
+  waste_carriers_engine:
+    copy_cards_forms:
+      new:
+        heading: "Order registration cards for %{reg_identifier}"
+        temp_cards_label: "Number of cards at £5 each"
+        next_button: "Continue"
```
Comment (cintamani) [inline config/locales/forms/copy_cards_forms/en.yml]:
```suggestion
        next_button: "Continue"
        check_the_address: "Check the address to post the cards to:"
```

**Pull request merged**